Pipfile

```diff
@@ -10,6 +10,8 @@ pytest = "*"
 [packages]
 django = "*"
```

Comment:
Tried to run your code and i failed on /bin/bash setup.sh
Please update you Pipfile.lock
remove/ copy Pipfile.lock
Run:
* pipenv clean
* pipenv lock